src/components/Book.css

```diff
@@ -0,0 +1,112 @@
+.book-container {
+  padding: 1.6rem 1.6rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px var(--grey-border);
+  background-color: var(--white);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book-details {
+  flex-basis: 50%;
+  flex-grow: 1;
+  flex-shrink: 1;
+}
+
+.book-details h2 {
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1.275rem;
+  font-weight: bold;
+  letter-spacing: -0.2px;
+  color: var(--black-two);
+  margin: 0.188rem 20.188rem 0 0;
+}
+
+.book-details h3 {
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  margin: 0 2.438rem 0 0;
+}
+
+.book-details p {
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  color: var(--cool-blue);
+  margin: 0.25rem 0 0;
+}
+
+.book-actions-container button {
```

Comment:
- [ ] [OPTIONAL] To improve even better I would suggest adding some hover effects to the buttons to make it perfect 🎯